code/practice/5.py

```
@@ -0,0 +1,36 @@
+# except:
+#     print("エラーが発生しました")
+# finally:
+#     print("処理終了")
+
+try:
+    with open("sample.txt", "r") as file:
+        content = file.read()
+        print(content)
+except FileNotFoundError:
+    print("ファイルが見つかりません")
+except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)